src/app/Notifications/Notifications.tsx

```diff
@@ -145,7 +146,8 @@ export class Notifications {
   }
 
   private isNetworkInfoNotification(n: Notification): boolean {
-    return (n.category === 'WsClientActivity' || n.category === 'TargetJvmDiscovery')
+    return (n.category === NotificationCategory.WsClientActivity
+      || n.category === NotificationCategory.JvmDiscovery)
```

Comment:
I think this function should at least be renamed, since JvmDiscovery doesn't seem like a "network info" or very related to WsClientActivity - but it would probably be better to split this into two functions.